.travis.yml

```diff
@@ -1,12 +1,27 @@
-git:
-  depth: 10
+language: node_js
+node_js:
+- '0.11'
+- '0.12'
+- '4.0'
+- '5.0'
+- '6.0'
+os:
+- linux
+- osx
+script: curl -s https://raw.githubusercontent.com/atom/ci/master/build-package.sh
```

Comment:
The node versions here are mostly to check if the building of the package works, which didn’t pass on 0.10. They’re probably overkill though, so we could as well remove some?